drivers/platform/stm32/stm32_xspi.c

```diff
@@ -5,36 +5,30 @@
 ********************************************************************************
  * Copyright 2024(c) Analog Devices, Inc.
  *
- * All rights reserved.
```

Comment:
please squash this commit into the commit that adds stm32_xspi implementation

since you add that implementation, add it with the proper license (3 clause BSD) from the start